README.md

```diff
@@ -27,6 +27,9 @@
     <li><a href="#on-a-container">On a container</a></li>
   </ul>
 </li>
+<li>
```

Comment:
Lets move this section to before "How to run"?

Reply:
done